### DISCUSSION TITLE
How to update SD.Next with new Python Directory?

### DISCUSSION THREAD

**Discussion by githubUser01946** [vladmandic/sdnext#2890] [Q&A]
Due to insufficient space on my C:/ I have installed Python on my D:/ which is the same drive as SD.Next.
When starting webui.bat it assumes that python is on C, how do I update it to see it on D instead?

Thanks

**Comment by vladmandic** [comment]
sdnext doesn't assume anything, it uses windows path. or you can set PYTHON env variable manually to set it to specific python (that can also be used if you have multiple python installs).

**Reply by githubUser01946** [reply]
I dont have any other python installed. I just installed it to D:/

During install of Python I put a tick in the box that mentioned PATH but as stated above it tris to load it from the usual install location on C, I guess ill need to do a search on how to update whatever needs updating so it knows where python actually is.

**Comment by githubUser01946** [comment]
Sorted. 

I updated the cfg in `D:\automatic\venv` to the location of Python, in this case it is now like this, I have only changed top line:

`home = D:\Python`

**Comment by SAC020** [comment]
Had a somewhat similar problem when updating the python version, but in this case I had to delete the "venv" folder and let it re-create itself by running webui.bat

Leaving this comment here as it might be useful to someone else in the same situation.
